modules/dr-restoring-cluster-state.adoc

```diff
@@ -186,6 +186,186 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
 
+. If you have lost the majority of your control plane hosts, leading to etcd quorum loss, delete and recreate the control plane machines (also known as the master machines), one by one. After these machines are recreated, a new revision is forced and etcd scales up automatically.
```

Comment:
I'm not sure if "If you have lost the majority of your control plane hosts, leading to etcd quorum loss," is necessary here. I believe that the only reason they should be doing this procedure at all is if they've lost the majority.

Also for "delete and recreate the control plane machines" - I believe they'd only want to do this for the ones that were lost (the non-recovery). I'm assuming we don't want them to recreate the recovery host machine, only the lost ones